lib/stripe/core_resources/charge.ex

```diff
@@ -105,14 +112,14 @@ defmodule Stripe.Charge do
     :amount,
```

Comment:
When I run against stripe-mock and turn on warnings  I get
```[warn]  Extra keys were received but ignored when converting Stripe.Charge: [:application_fee, :payment_method, :payment_method_details]```

The first one was changed here, but the api still returns it. Should it not still be included?

The other two are also missing, but might be for later PR?

Reply:
Ready for your re-review!